### PR TITLE
DB Manager: remove semicolon from SQL window queries

### PR DESCRIPTION
**Pull request by elpaso**
Partial fix for #36205

### CODE REVIEW

File: python/plugins/db_manager/dlg_sql_window.py [hunk 1]
```diff
@@ -653,7 +653,9 @@ def _getSqlQuery(self):
         return sql
 
     def _getExecutableSqlQuery(self):
-        sql = self._getSqlQuery()
+        sql = self._getSqlQuery().strip()
```
Comment (m-kuhn) [inline python/plugins/db_manager/dlg_sql_window.py:656]:
```suggestion
        sql = self._getSqlQuery().strip(' \n\r;')
```

Would that help when there's a trailing space/newline?

e.g.

```
select *
from x;

```

or do the check on the uncommented sql (which removes newlines)

Comment (elpaso) [reply python/plugins/db_manager/dlg_sql_window.py:656]: I'm fixing another issue here. Please file another ticket.

Comment (m-kuhn) [reply python/plugins/db_manager/dlg_sql_window.py:656]: My proposal aimed at a more generic approach for *exactly* this issue and nothing else.
I'll open a follow up pull request.

Comment (elpaso) [reply python/plugins/db_manager/dlg_sql_window.py:656]: Ok, thank you!

**Pull request merged**